rmgpy/statmech/conformer.pyx

```diff
@@ -262,6 +262,8 @@ cdef class Conformer(RMGObject):
                 N += 1
             elif type(mode) == SphericalTopRotor:
                 N += 3
+            elif hasattr(mode,'dof'):
```

Comment:
please add a space between `mode` and `'dof'`

Reply:
done